packages/lambda/package.json

```diff
@@ -32,5 +31,8 @@
     "serverless-http": "^2.3.0",
     "strip-indent": "^3.0.0"
   },
+  "devDependencies": {
+    "aws-sdk": "^2.546.0"
```

Comment:
How is this supposed to work? The deployment `hops lambda deploy` needs the `aws-sdk` and it won't be installed when it is a peer dependency.

Reply:
My bad. Thought this was ending up in the bundle, which is not the case. Will remove that commit.